tests/integration/telemetry/flags.go

```diff
@@ -16,30 +16,18 @@
 package telemetry
 
 import (
-	"testing"
-
-	"istio.io/istio/pkg/test/framework"
-	"istio.io/istio/pkg/test/framework/components/istio"
-	"istio.io/istio/pkg/test/framework/components/istio/ingress"
-	"istio.io/istio/pkg/test/framework/label"
-	"istio.io/istio/pkg/test/framework/resource"
+	"flag"
+	"time"
 )
 
 var (
-	i    istio.Instance
-	ingr []ingress.Instance
+	// RetryDelay is the retry delay used in tests.
+	RetryDelay time.Duration
+	// RetryTimeout is the retry timeout used in tests.
+	RetryTimeout time.Duration
 )
 
-func TestMain(m *testing.M) {
-	framework.
-		NewSuite(m).
-		Label(label.CustomSetup).
-		Setup(istio.Setup(&i, nil)).
-		Setup(func(ctx resource.Context) (err error) {
-			for _, cl := range ctx.Clusters() {
-				ingr = append(ingr, i.IngressFor(cl))
-			}
-			return nil
-		}).
-		Run()
+func init() {
+	flag.DurationVar(&RetryDelay, "istio.test.telemetry.retryDelay", time.Second*3, "Default retry delay used in tests")
```

Comment:
I think you also have to update https://github.com/istio/istio/blob/master/pkg/test/framework/initstatics.go to import this package so that the flags are registered.